WTWSMS/history/characters/hindustani.txt

```diff
@@ -218,7 +218,7 @@
 	}
 }
 190205 = {
-	name="Eusabic" # Theodoric's Mother
+	name="Ereleuva" # Theodoric's Mother
```

Comment:
She was baptised in the name of Eusebia, which is Eusabic in Gothic if I get it correctly.

Reply:
I found that too, but didn't find the Gothic form. Now that I know what language it is, I confirm that Eusabic is a good form. I will revert it.

Reply:
Done.